### PR TITLE
updated the readme follow step to contribute

### PR DESCRIPTION
**Pull request by DreXuri**
this step will help others contribute to the project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,51 @@
...
+
+....
+flutter run 
+....
+
+5. make changes
+
+6. create a branch
+
+7. cheackout to your branck
```
Comment (moak13) [inline README.md:45]: kindly check the spelling of your `branch`.

**Review by moak13 — CHANGES_REQUESTED** [review]
Nice work, kindly see my concerns below.

File: README.md (hunk 1 above)
Comment (DreXuri) [reply README.md:45]: ok, I'll correct an revert

Comment (DreXuri) [reply README.md]: i have addressed your comment

File: README.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ flutter run
 
 6. create a branch
 
-7. cheackout to your branck
+7. cheackout to your branch
```
Comment (moak13) [inline README.md:45]: your spelling of checkout too is not correct, I wonder how I missed that.

**Review by moak13 — CHANGES_REQUESTED** [review]
kindly treat below

File: README.md [hunk 3]
```diff
@@ -1,3 +1,47 @@
 # receipe_app
 
-A new Flutter project.
+How to contribute to this open source project
+ Prerequisites
+* You will need to have flutter set up on your system  [https://docs.flutter.dev/get-started/install]
+* You Should know this project was build using stacked as a statemanagement tool  check this for info on stacked[https://stacked.filledstacks.com/docs/getting-started/overview/]
```
Comment (moak13) [inline README.md:6]: `build` to `built`, `statemanagement` to `state management`

**Review by moak13 — CHANGES_REQUESTED** [review]
last bit

File: README.md (hunk 3 above)
Comment (DreXuri) [reply README.md]: ok, effected.
you can check.\

Comment (moak13) [reply README.md]: missed the first one

Comment (DreXuri) [reply README.md]: okay

**Comment by moak13** [comment]
LGTM! Well done.

**Pull request merged**